github-issues.go

```diff
@@ -195,6 +199,10 @@ func populateIssues(url string) {
 			eachGitIssue.Title = elem.Title
 			eachGitIssue.Repo = elem.RepositoryURL
 			eachGitIssue.Link = elem.HTMLURL
+			eachGitIssue.CreatedAt = elem.CreatedAt.Format(layout)
+			eachGitIssue.UpdatedAt = elem.UpdatedAt.Format(layout)
+			delta := elem.UpdatedAt.Sub(elem.CreatedAt)
+			eachGitIssue.Age = (int)(delta.Hours())
```

Comment:
Starting with like this 

```
s/GitPRs/GithubPRs
```

```
s/Repoissue/RepoIssue
```

Each distinct word in itself is capitalized. Next question is do they need to be exposed i.e in Go keeping a struct as 'CanonicalStruct' v/s 'canonicalStruct' the latter is preferred here since we are not making this a library and neither will be used by anyone outside of this project.

Also we need to have a valid reason to avoid using libraries such as this https://github.com/google/go-github because we are doing low level work here and should be avoided. 

https://github.com/google/go-github/blob/master/github/pulls.go#L100 can give you nice paginated output as well.  Maintaining this struct is harder IMHO, if Github changes the API we have to keep managing this. 

Similarly with this https://godoc.org/github.com/google/go-github/github#IssuesService.ListByRepo

Your overall work will be reduced and code will be simple/clean.  

Now to `main.go`  has multiple things

```go
var token = ""  --> This purpose is not clear

// TomlConfig - holds all the repo names
type tomlConfig struct {
        RepoNames []string `toml:"repoNames"`
}

var config tomlConfig

func main() {

        token = os.Getenv("GIT_TOKEN")
        fmt.Println(token) --> Frivolous printing not needed, avoid debug statements in project repo.
        if token == "" {
                fmt.Println("Git Token is not set")
                os.Exit(101) --> why 101 ? why not just 1? This could be have been simply `exitOnErr(errors.New("Github token is not set"))` 
--> stray new line. 
        }
        if _, err := toml.DecodeFile("repo.toml", &config); err != nil {
                exitOnErr(err)
        }

        auth() --> too generic name doesn't need to be a function IMHO, you can verify the token right here.  If we move to go-github all this problem also goes away. 

        http.HandleFunc("/getIssues", getIssues)
        http.HandleFunc("/getPRs", getPRs)
        fs := http.FileServer(http.Dir("static"))
        http.Handle("/", fs)

        http.ListenAndServe(":7000", nil)
}
```

The reason to bring this up is because since this is a public facing project we need to keep this upto the quality of all our projects and consistent in style, comments.  You can look at https://github.com/golang/go/wiki/CodeReviewComments this document which describes some interesting things about Go idiomatic style.